### PR TITLE
Updated CRD removal section for Serverless

### PR DESCRIPTION
**Pull request by abrennan89**
@cardil if this looks OK to you can  get a `QE approved` comment please?
Thanks.

### CODE REVIEW

**Comment by cardil** [comment]
LGTM. QE approved

File: serverless/installing_serverless/removing-openshift-serverless.adoc [hunk 1]
```diff
@@ -22,32 +22,23 @@ include::modules/serverless-uninstalling-knative-serving.adoc[leveloffset=+1]
...
+[id="deleting-serverless-CRDs"]
+== Deleting {ServerlessProductName} CRDs
 
-After uninstalling the {ServerlessOperatorName}, the Operator CRDs and API services remain on the cluster. Use this procedure to completely uninstall the remaining components.
+After uninstalling the {ServerlessProductName}, the Operator and API CRDs remain on the cluster.
+You can use the following procedure to remove the remaining CRDs.
+
+[IMPORTANT]
+====
+Removing the Operator and API CRDs will remove all resources that were defined using them, including Knative services.
```
Comment (kalexand-rh) [inline serverless/installing_serverless/removing-openshift-serverless.adoc:33]: s/will remove/also removes

File: serverless/installing_serverless/removing-openshift-serverless.adoc [hunk 2]
```diff
@@ -22,32 +22,23 @@ include::modules/serverless-uninstalling-knative-serving.adoc[leveloffset=+1]
...
+====
 
 .Prerequisites
-*  You have uninstalled Knative Serving and removed the {ServerlessOperatorName} using the previous procedure.
+*  You have uninstalled Knative Serving and have removed the {ServerlessOperatorName}.
```
Comment (kalexand-rh) [inline serverless/installing_serverless/removing-openshift-serverless.adoc:37]: I'd remove both instance of "have."

**Review by kalexand-rh — COMMENTED** [review]
Two picks and it LGTM

**Pull request merged**